src/components/templates/ErrorPages/Template/Template.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { IconDefinition } from '@fortawesome/fontawesome-common-types';
+
+import FlexContainer from '../../../layout/FlexContainer/FlexContainer';
+import FlexRow from '../../../layout/FlexRow/FlexRow';
+import FlexCol from '../../../layout/FlexCol/FlexCol';
+import { ProductTemplateCard } from '../../ProductTemplate/ProductTemplateCard/ProductTemplateCard';
+
+export interface ErrorTemplateProps {
+  icon?: IconDefinition;
+}
+
+const ErrorTemplate: React.FC = ({ children }) => {
+  return (
+    <FlexContainer data-automation="ZA.FiveHundredTemplate" gutter={0}>
```

Comment:
Should this be `ZA.FiveHundredTemplate` as it will also appear for the other templates>